lib/library.go

```diff
@@ -197,15 +197,15 @@ func Logout() *C.char {
 //CompleteTransaction instructs backend to complete sending of a given transaction
 //export CompleteTransaction
 func CompleteTransaction(id, password *C.char) *C.char {
-	txHash, err := statusAPI.CompleteTransaction(common.QueuedTxID(C.GoString(id)), C.GoString(password))
+	txHash, err := statusAPI.CompleteTransaction(utils.QueuedTxID(C.GoString(id)), C.GoString(password))
 
 	errString := ""
 	if err != nil {
 		fmt.Fprintln(os.Stderr, err)
 		errString = err.Error()
 	}
 
-	out := common.CompleteTransactionResult{
+	out := utils.CompleteTransactionResult{
```

Comment:
I am not sure if `CompleteTransactionResult` should be declared in `geth/transactions`. It's used only in `lib` package, right?